### ISSUE TITLE
Git tag failing with single quotes in the body of the annotation

### ISSUE THREAD

**Issue by piotrb** [release-it/release-it#577]
This is related and a followup to #576 

It looks like the new escaping code is failing to escape single quotes properly now .. 

```
  release-it:shell    `git tag --annotate --message='Release 1.6.0\n\n* Wait longer, but also more aggressively try to acquire the lock (2d52b78f)\n* Add key to redlock errors (a551346b)\n* Sanity checks around update throttler errors (c36f096e)\n* Increase default page size (fe80ec5a)\n* Add version_key to all resources (59106bfb)\n* Tweak jsonapi (84b3bd4b)\n* Fix? (b3169139)\n* More loading tweaks (30ab8b29)\n* Fix a bunch of loading issues .. again (82c7ca53)\n* Handle api errors better (77372afe)\n* Move some common typed out of a d.ts file (030e50c7)\n* Fix line numbers in sourcemaps (f7ac1aa4)\n* Extend the api type map to give a bunch more info about relations and fk's (6a2b170a)\n* Fixes (5ed87a80)\n* Fix to missing provide (12f48b87)\n* Lint tweak (3d132e0c)\n* Tweaks to building (6a8ab48d)\n* Remove legacy api controllers (f157d575)\n* Add check for yarn dupes (0b253b0d)\n* Remove the remainder of code from frontend (bc8cfab7)\n* Convert executions and execution info to new model code (8d76df82)\n* Add full type definition of the "data" field on executions (94f903d2)\n* Tweaks to Deploys view (2c748c27)\n* Add eslint validations for disable comments, and fixed the issues (6a6b174c)\n* Convert Deploys list ui to new model code (6c1fc350)\n* Filter readonly attributes from create/update calls (14473ca3)\n* Add support for sending cable updates only after the transaction has commited (1a54b965)\n* Add a few new decorators to wrap around the pattern of getting related records (310b449d)\n* Add only_deleted filter to deploys (78eb26e3)\n* Make askActual properly toggle its return type based on the boolean arg (d4cd17e7)\n* Add proper type to Deploys tag field (334af2bf)\n* Disable checking for duplicate class members to support TS overloads (eae0144f)\n* Refactor out the colgroup size pattern (cc659d81)\n* Created Select All checkbox component (033a1665)\n* Extract behavior of "manually selectabe table" (eb181a33)'  1.6.0`,
```

Fails with:

```
  release-it:shell   stderr:
  release-it:shell    "/bin/sh: -c: line 14: syntax error near unexpected token `('\n/bin/sh: -c: line 14: `* Extend the api type map to give a bunch more info about relations and fk's (6a2b170a)'\n" } +5s
```

And if you look closely the single quote is not being escaped but its being used as the outer wrapper for this string.

**Comment by webpro** [comment]
Ah, my bad, only the commands themselves got quoted properly, but not the replacements (e.g. the actual value of `${changelog}`). You can try in the `next` release (`12.5.0-next.1`), just `npm install release-it@next` again.

**Comment by webpro** [comment]
Should be fixed in [v12.5.0](https://github.com/release-it/release-it/releases/tag/12.5.0)
